mecademic_robot_node/src/mecademic_robot_driver/mecademic_robot_driver.py

```diff
@@ -168,16 +168,16 @@ def __del__(self):
         """Deconstructor for the Mecademic Robot ROS driver
         Deactivates the robot and closes socket connection with the robot
         """
-        self.robot.Deactivate()
-        self.robot.Disconnect()
-        self.feedback.Disconnect()
+        self.robot.DeactivateRobot()
+        self.robot.disconnect()
+        self.feedback.disconnect()
 
 if __name__ == "__main__":
-    robot = RobotController('192.168.0.100')
```

Comment:
The change from 192.168.0.100 to 192.168.1.9 is our change, the Mecademic default is still 192.168.0.100. So let's change that back.